pykwalify/rule.py

```diff
@@ -408,6 +418,13 @@ def init_allow_empty_map(self, v, rule, path):
 
         self.allowempty_map = v
 
+    def init_allow_none(self, v, rule, path):
+        log.debug(u"Init allow none value: %s", path)
+        log.debug(u"Type: %s : %s", v, rule)
+
+        self.allownone = v
```

Comment:
`allownone` should validate that what i write in the schema is `True` or `False`, all other values should not be allowed because the flag is a boolean flag.